src/Managers/MediaManager.php

```diff
@@ -92,13 +96,15 @@ public function uploadFromString(
      * @param  string  $string
      * @param  string|null  $name
      * @param  string  $collection
+     * @param  array  $customProperties
      * @return \Spatie\MediaLibrary\MediaCollections\Models\Media
      */
     public function uploadFromBase64(
         HasMedia $model,
         string $string,
         $name = null,
-        string $collection = 'default'
+        string $collection = 'default',
+        array $customProperties = []
```

Comment:
Could you validate why the actions is not passing anymore?
```suggestion
        array $customProperties = [],
```